packages/extension-ui/src/components/SplashHandler.tsx

```diff
@@ -57,15 +84,15 @@ function SplashHandler({ children, className }: SplashHandlerProps): React.React
             }}
           >
             <Video
-              onEnded={setSplashState}
-              onStarted={setContentVisible}
+              onEnded={onVideoEnd}
+              onStarted={onVideoPlay}
```

Comment:
I think this would convey as much information, yet with less verbosity:
```suggestion
              onEnded={() => setIsContentVisible(true)}
              onStarted={() => setIsSplashOn(false)}
```

Reply:
fair enough :+1: